stubs/default/routes/auth.php

```diff
@@ -31,7 +32,7 @@
                 ->name('password.reset');
 
     Route::post('reset-password', [NewPasswordController::class, 'store'])
-                ->name('password.update');
+                ->name('password.store');
```

Comment:
This was renamed because the `update` method on the new `PasswordController` is more deserving of this name.